core/src/main/java/org/apache/carbondata/core/datamap/TableDataMap.java

```diff
@@ -437,8 +430,8 @@ private int getNumOfThreadsForPruning() {
    * Clear only the datamaps of the segments
    * @param segments
    */
-  public void clear(List<Segment> segments) {
-    for (Segment segment: segments) {
+  public void clear(List<String> segments) {
```

Comment:
what is the string in list, please write comment

Reply:
done